openslides/core/static/js/core/projector.js

```diff
@@ -160,6 +161,28 @@ angular.module('OpenSlidesApp.core.projector', ['OpenSlidesApp.core'])
             });
         };
 
+        var STEPS = 5;
```

Comment:
Please add a short comment or docstring about what this is doing (small scroll animation ...)